File: test/fixtures/php/post_quotes_inside_data.php
<?php
$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, 'http://localhost:28139');
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'POST');
curl_setopt($ch, CURLOPT_HTTPHEADER, [
    'Content-Type: application/x-www-form-urlencoded',
]);
curl_setopt($ch, CURLOPT_POSTFIELDS, 'field=don%27t%20you%20like%20quotes');

$response = curl_exec($ch);

curl_close($ch);

File: test/fixtures/r/post_quotes_inside_data.r
library(httr)

headers = c(
  `Content-Type` = "application/x-www-form-urlencoded"
)

data = list(
  field = "don't you like quotes"
)

res <- httr::POST(url = "http://localhost:28139", httr::add_headers(.headers=headers), body = data, encode = "form")
